internal/process/datacenter.go

```diff
@@ -91,3 +104,43 @@ func createDatacenterSamples(config *load.Config) {
 
 	}
 }
+
+func processEvent(ed *events.EventDispacher, entity *integration.Entity) error {
+
+	if ed == nil {
+		return fmt.Errorf("not expecting empty EventDispacher")
+	}
+	for _, be := range ed.Events {
+		e := be.GetEvent()
+
+		ev := &eventSDK.Event{
+			Summary:  e.FullFormattedMessage,
+			Category: "vSphereEvent",
+			Attributes: map[string]interface{}{
+				"vSphereEvent.userName": e.UserName,
+				"vSphereEvent.date":     e.CreatedTime.Format(time.RFC1123),
+			},
+		}
+		if e.Vm != nil {
+			ev.Attributes["vSphereEvent.vm"] = e.Vm.Name
+		}
+		if e.Host != nil {
+			ev.Attributes["vSphereEvent.host"] = e.Host.Name
+		}
+		if e.Datacenter != nil {
+			ev.Attributes["vSphereEvent.datacenter"] = e.Datacenter.Name
+		}
+		if e.ComputeResource != nil {
+			ev.Attributes["vSphereEvent.computeResource"] = e.ComputeResource.Name
+		}
+		if e.Ds != nil {
+			ev.Attributes["vSphereEvent.datastore"] = e.Ds.Name
+		}
+		err := entity.AddEvent(ev)
```

Comment:
is there any events for cluster?

Reply:
it is the compute resource, that it can be a host or a cluster, but the Network info was missing. I add it!